js/client-chat.js

```diff
@@ -958,6 +958,13 @@
 					});
 					break;
 
+				case 'remove':
+					if (app.user.get('isStaff')) break;
+					$('.chatmessage-' + toId(row[1])).each(function() {
+						$(this).remove();
+					});
```

Comment:
Wow, you really don't know how to jQuery, do you?

That is literally just a slow and unreadable way to write:

    $('.chatmessage-' + toId(row[1])).remove()